src/app/shared/object-list/my-dspace-result-list-element/claimed-search-result/claimed-declined-task-search-result/claimed-declined-task-search-result-list-element.component.html

```diff
@@ -4,7 +4,6 @@ <h4 class="alert-heading mb-0">{{ 'claimed-declined-task-search-result-list-elem
     <ds-themed-item-list-preview *ngIf="workflowitem"
                                  [item]="(workflowitem?.item | async)?.payload"
                                  [object]="object"
-                                 [status]="status"
```

Comment:
here the context seems to be not passed anymore, is it correct ?

Reply:
It looks like @atarix83 's prior question was not answered here.  I have the same question:
> here the context seems to be not passed anymore, is it correct ?